lib/errors.coffee

```diff
@@ -26,4 +30,9 @@ exports.handle = (error) ->
 		message = error.stack
 
 	patterns.printErrorMessage(message)
-	process.exit(error.exitCode or 1)
+
+	captureException(error)
+	.timeout(1000)
+	.catch(->)
```

Comment:
This is an odd thing to complain about. Yes, houndbot, that's exactly what I intended. How do I do this without annoying the bot?